torch/_dynamo/variables/builder.py

```diff
@@ -1022,11 +1037,21 @@ def wrap_unspecialized_primitive(self, value):
         if self.name in self.tx.output.unspec_variable_map:
             return self.tx.output.unspec_variable_map[self.name]
         else:
+            shape_env = self.tx.output.shape_env
+            if torch._dynamo.config.force_unspec_int_unbacked_size_like and isinstance(
+                value, int
+            ):
+                wrapped_value = shape_env.create_unbacked_symint()
+                _constrain_range_for_size(wrapped_value)
```

Comment:
Can we use constrain_as_size instead?

Reply:
This fails with `torch._dynamo.exc.InternalTorchDynamoError: tried to get Int out of SymInt`